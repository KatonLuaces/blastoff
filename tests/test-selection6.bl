A = [1,2,3;4,5,6;7,8,9];
B = A[[0;2], [0;2], 1, 1];
v = [1;2;3;4];
u = v[[0;2;3]];
print(B);
print(u);
