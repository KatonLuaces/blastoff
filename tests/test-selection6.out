1 3
7 9
1
3
4